src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php

```diff
@@ -81,7 +81,7 @@ class CardSchemeValidator extends ConstraintValidator
         ),
         // All Visa card numbers start with a 4. New cards have 16 digits. Old cards have 13.
```

Comment:
Comment should be removed or updated as well?

Reply:
comment updated